go.mod

```diff
@@ -397,19 +406,6 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-require github.com/netsampler/goflow2 v1.1.1-0.20220825033856-d6caeaacddbb
-
-require (
-	github.com/DataDog/aptly v1.5.0 // indirect
-	github.com/agnivade/levenshtein v1.0.1 // indirect
-	github.com/cavaliergopher/grab/v3 v3.0.1 // indirect
-	github.com/dgraph-io/ristretto v0.1.0 // indirect
-	github.com/libp2p/go-reuseport v0.1.0 // indirect
-	github.com/vektah/gqlparser/v2 v2.4.6 // indirect
-	gomodules.xyz/jsonpatch/v2 v2.2.0 // indirect
-	k8s.io/apiextensions-apiserver v0.23.5 // indirect
-)
-
```

Comment:
??

Reply:
Hmm I'm not sure, it might have come about from a `go mod tidy` or perhaps from an oddity when merging changes happening in main here?